gear-test/src/sample.rs

```diff
@@ -97,7 +104,7 @@ pub struct Expectation {
     pub allow_error: Option<bool>,
     /// Expected active programs (not failed in the init) ids
     #[serde(rename = "programs")]
-    pub active_programs: Option<Vec<ChainAddress>>,
+    pub programs: Option<Vec<ChainProgram>>,
```

Comment:
Also I would like to see here a flag `only` that we discussed with you some time ago:
true will mean that this is the only existing programs
false will mean checking that specified programs have concrete state